### ISSUE TITLE
About model installation in pymt

### ISSUE THREAD

**Issue by XianaZhang** [csdms/pymt#106]
Hi guys,
I came across a cofused problem when I install sedflux model using Anaconda prompt.  The solving environment was always failed to be set. And this problem is the same with other models‘ installation, such as Hydrotrend. As shown below:

(pymt) C:\Users\张明杰>conda install pymt_hydrotrend -c conda-forge
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): failed

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://conda.anaconda.org/conda-forge/win-64/repodata.json>

Hope to get your reply as soon as possible.

**Comment by gregtucker** [comment]
I just tried reproducing this error, but the installation worked fine for me. @mcflugen any suggestions for @XianaZhang ?

**Comment by mcflugen** [comment]
@XianaZhang The HTTP error may just be due to a network problem with *conda-forge*. However, a bigger problem is that neither Hydrotrend nor Sedflux has been built for Windows.

This is a problem that we would like to eventually fix but, until then, if you have access to either a Linux or Mac machine you should be able to install *pymt_hydrotrend*. Is that a possibility?

**Comment by XianaZhang** [comment]
Oh! Thanks! So is the Sedflux software is also built under Linux or Mac machine ?------------------ 原始邮件 ------------------
发件人: "Eric Hutton"<notifications@github.com>
发送时间: 2020年2月4日(星期二) 凌晨5:10
收件人: "csdms/pymt"<pymt@noreply.github.com>;
抄送: "XianaZhang"<2738879286@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [csdms/pymt] About model installation in pymt (#106)



@XianaZhang The HTTP error may just be due to a network problem with conda-forge. However, a bigger problem is that neither Hydrotrend nor Sedflux has been built for Windows.
 
This is a problem that we would like to eventually fix but, until then, if you have access to either a Linux or Mac machine you should be able to install pymt_hydrotrend. Is that a possibility?
 
&mdash;

**Comment by mcflugen** [comment]
@XianaZhang Correct. I'm sure it could be built under Windows but we have not done that yet.
